docs/src/_reference/command-line-interface.md

```diff
@@ -701,13 +701,12 @@ Note that if no environment is active, `meltano run` _does not_ generate a state
 
 ## `job`
 
-Use the `job` command to define one more related tasks. A job can contain a single task, or many tasks. Today all tasks
-are run sequentially. You can run a specified job by passing the job name as an argument to `meltano run`.
+Use the `job` command to define one or more related tasks. A job can contain a single task, or many tasks. Today all tasks
```

Comment:
"define one or more related tasks" makes it seem like this would add multiple task definitions to `meltano.yml`, but it adds a single job definition _consisting of_ one or more tasks that will be run sequentially, right? I think it's important to state the relation of jobs to tasks to blocks to plugin (commands) clearly, and now I'm a bit confused.

Also, I don't think "related" here helps, they're not just "related plugins", they're steps in a pipeline.